scripts/linters.sh

```diff
@@ -52,3 +52,29 @@ find ./ \
      -and -not -wholename "./tests/func/tmp/*" \
      -and -name "*.yml" -print0 \
     | xargs -0 ansible-lint
+
```

Comment:
Not a blocker, but next time we should be putting such things into their own commits. (Not necessarily their own PRs, but at least commit separation is good.)

Reply:
Moving this after is ok before merge

Reply:
done